volume/local/local.go

```diff
@@ -363,6 +365,17 @@ func (v *localVolume) unmount() error {
 	return nil
 }
 
+func (v *localVolume) CreatedAt() (time.Time, error) {
+	fileInfo, err := os.Stat(v.path)
```

Comment:
Nevermind I guess it makes sense especially for volumes that pre-exist in the driver and just get "created" in docker but actually created before.